dhall/tests/Dhall/Test/Format.hs

```diff
@@ -69,3 +80,13 @@ formatTest characterSet prefix =
                 <> "Actual   (show): " <> show actualText <> "\n"
 
         Tasty.HUnit.assertBool message (actualText == expectedText)
+
+idempotenceTest :: TestTree
```

Comment:
I think I'd slightly prefer to keep the property tests together in `D.T.QuickCheck`.

Reply:
Moved